Files.Launcher/Win32API.cs

```diff
@@ -165,5 +97,34 @@ public static void UnlockBitlockerDrive(string drive, string password)
                 // If user cancels UAC
             }
         }
+
+        public static void Foreground(this Process process)
```

Comment:
Now window is opening foreground, but it seems like this window is not focused(i can't write command in terminal) and i need to click to focus

Reply:
Ok I'm removing the not working code.
- I'm not sure why `SetForegroundWindow()` is not working, according to the docs you are allowed to bring to the foreground a process you started yourself.
- `SetFocus` and `SetActiveWindow` do not work as they require `AttachThreadInput` to succeed (and it's not)
- `AttachThreadInput` does not succeed: in the above that's normal because the thread does not have a message queue